ilps22qs_reg.h

```diff
@@ -496,9 +496,8 @@ int32_t ilps22qs_id_get(const stmdev_ctx_t *ctx, ilps22qs_id_t *val);
 
 typedef enum
 {
-  ILPS22QS_SEL_BY_HW      = 0x00, /* bus mode select by HW (SPI 3W disable) */
-  ILPS22QS_SPI_3W         = 0x03, /* bus mode select by HW (SPI 3W disable) */
-  ILPS22QS_SPI_4W         = 0x02, /* bus mode select by HW (SPI 3W disable) */
```

Comment:
I was expecting only one line changed. I think it would be a better commit, easier to understand. Can you possibly do that?? (i.e. leaving first two lines equal, and just delete the last one)

Reply:
There are effectively two changes:
- line related to SPI 4W - removed
- line related to SPI 3W - changed comment from SPI 3W **disable** to **enable**